src/openforms/formio/service.py

```diff
@@ -45,7 +45,7 @@
 ]
 
 
-def format_value(component: Component, value: Any, as_html: bool = False):
+def format_value(component: Component, value: Any, *, as_html: bool = False):
```

Comment:
Is that necessary here?I mean is it not obvious that we have kwargs after `value`?
i guess it's a good practice to do it this way?

Reply:
Yes: https://adamj.eu/tech/2021/07/10/python-type-hints-how-to-avoid-the-boolean-trap/ :)